core/src/main/kotlin/SlashCommands.kt

```diff
@@ -112,6 +115,40 @@ class SlashCommands(
             emit(GlobalApplicationCommand(data, service))
         }
     }
+
+    suspend fun getGuildApplicationCommandPermissions(
+            applicationId: Snowflake,
+            guildId: Snowflake,
+    ): DiscordApplicationCommandPermissions =
+            service.getGuildApplicationCommandPermissions(applicationId, guildId)
+
```

Comment:
You shouldn't be returning DiscordX contents. Core entities must be added for those along side data objects

Reply:
This remains a rest entity, a core entity should be returned

Reply:
Never mind, GH showing an old revision